src/main/java/domain/lotto/LottoService.java

```diff
@@ -0,0 +1,66 @@
+package domain.lotto;
+
+import domain.lottoTicket.LottoTicket;
+import domain.lottoTicket.LottoTicketService;
+import domain.lottoWinningStatistics.LottoWinningStatistics;
+import domain.lottoWinningStatistics.LottoWinningStatisticsService;
+import java.util.ArrayList;
+import java.util.List;
+import view.InputView;
+import view.OutputView;
+
+public class LottoService {
+    private final int LOTTO_PRICE = 1000;
+    private final InputView inputView;
+    private final  OutputView outputView;
+    public LottoService(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        int buyingMoney = inputView.readMoney();
+        Lotto lotto = new Lotto(buyingMoney, countLottoTickets(buyingMoney));
+        List<LottoTicket> lottoTickets = generateLottoTicketList(lotto);
+        printLottoTickets(lotto, lottoTickets);
+        drawLottoWinning(lotto, lottoTickets);
+    }
+
+    public int countLottoTickets(int money) {
+        int lottoTicketCount = (int) money / LOTTO_PRICE;
+
+        return lottoTicketCount;
+    }
+
+    public List<LottoTicket> generateLottoTicketList(Lotto lotto) {
+        LottoTicketService lottoTicketService = new LottoTicketService();
+
+        List<LottoTicket> lottoTicketList = new ArrayList<>();
+
+        for(int i = 0; i < lotto.getLottoTicketCount(); i++) {
+            LottoTicket lottoTicket = new LottoTicket(lottoTicketService.generateLottoTicket());
+            lottoTicketList.add(lottoTicket);
+        }
+
+        return lottoTicketList;
+    }
+
+    public void printLottoTickets(Lotto lotto, List<LottoTicket> lottoTickets) {
+        outputView.writeLottoTickets(lotto, lottoTickets);
+    }
+
+    public void drawLottoWinning(Lotto lotto, List<LottoTicket> lottoTickets) {
+        List<Integer> winningNumbers = inputView.readWinningNumber();
+        LottoWinningStatisticsService lottoWinningStatisticsService = new LottoWinningStatisticsService();
```

Comment:
```suggestion
    private final LottoWinningStatisticsService lottoWinningStatisticsService;
    
    public void drawLottoWinning(Lotto lotto, List<LottoTicket> lottoTickets) {
        List<Integer> winningNumbers = inputView.readWinningNumber();
```

이것도 의존성이 외부에서 주입되어 더 유연하고 테스트하기 쉬울 것 같아요!